graph/clients/listenNoteClient.js

```diff
@@ -22,9 +22,10 @@ export const getEpisode = async (episode_id)=>{
 
 export const search = async (query, sort_by_date=0, type='podcast', offset=0, len_min=10, len_max=100,
   genre_ids=[68, 82], published_before=Date.now(), published_after=0,
-  only_in=['title', 'description'], language='English', safe_mode=1)=>{
+  only_in=['title', 'description','author'], language='English', safe_mode=1)=>{
+  // return callListenNote(`/search?q=${query}&sort_by_date=${sort_by_date}&type=${type}&offset=${offset}&len_min=${len_min}&len_max=${len_max}&genre_ids=${genre_ids.join('%2C')}&published_before=${published_before}&published_after=${published_after}&only_in=${only_in.join('%2C')}&language=${language}&safe_mode=${safe_mode}`)
```

Comment:
Let's remove this commented code and remove unused function parameters

Reply:
that is for convenience sake , so later it will be easy to recover ,because anyway we are going to end up with those params